libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -498,12 +498,24 @@ void AP_MotorsMatrix::setup_motors(motor_frame_class frame_class, motor_frame_ty
                     add_motor(AP_MOTORS_MOT_3,   0, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  1);
                     add_motor(AP_MOTORS_MOT_4, 180, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  3);
                     break;
+                case MOTOR_FRAME_TYPE_NYT_PLUS:
```

Comment:
Maybe a #if round this for Plane, could be exciting to try this on copter.

Reply:
that reminds me that Buzz was talking about having a branch which used tilt servo(s) instead of torque for yaw
I guess the sanity check would more properly be done as a pre-arm test; will look to see if there are any similar checks already.

Reply:
went with the #if approach, thanks